File: CodeFreak1/ViewModel/MessageCompleteViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class MessageCompleteViewModel
    {
        public MessageViewModel message { get; set; }
        public UsersViewModel sender { get; set; }
        public UsersViewModel reciever { get; set; }
        public string currentUserId { get; set; }

    }
}

File: CodeFreak1/ClientApp/src/app/chat/Dtos/Message-complete-view-model.ts
import { MessageViewModel } from "./MessageViewModel";
import { UsersViewModel } from "../../Security/Dtos/users-view-model";

export class MessageCompleteViewModel
    {
        public  message: MessageViewModel;
        public  sender: UsersViewModel;
        public reciever: UsersViewModel;
        public currentUserId: string;

        constructor() {
            this.message = new MessageViewModel();
            this.sender = new UsersViewModel();
            this.reciever = new UsersViewModel();
          }

    }